src/styles/blocks/_product-card.scss

```diff
@@ -0,0 +1,82 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 14px;
+  margin-bottom: 50px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 45px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    color: #000;
+    margin-bottom: 6px;
+  }
+
+  &__subtitle {
+    margin-bottom: 15px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
```

Comment:
if you have just 1 font family in project, you can replace it to normalize/reset file and don't repeat yourself